cypress/e2e/settings/data-export/transformation-form.cy.js

```diff
@@ -0,0 +1,82 @@
+import testTypes from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import SettingsPane from '../../../support/fragments/settings/settingsPane';
+import Users from '../../../support/fragments/users/users';
+import InteractorsTools from '../../../support/utils/interactorsTools';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import ExportFieldMappingProfiles from '../../../support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles';
+import ExportNewFieldMappingProfile from '../../../support/fragments/data-export/exportMappingProfile/exportNewFieldMappingProfile';
+import ModalSelectTransformations from '../../../support/fragments/data-export/exportMappingProfile/modalSelectTransformations';
+
+let user;
+let fieldMappingProfileName = `fieldMappingProfile${getRandomPostfix()}`;
+const newTransformationCalloutMessage = '1 transformation has been successfully added';
+const newFieldMappingProfileCalloutMessage = `The field mapping profile ${fieldMappingProfileName} has been successfully created`;
+
+describe('settings: data-export', () => {
+    before('creating user and navigating to settings', () => {
+        cy.createTempUser([
+            permissions.inventoryAll.gui,
+            permissions.dataExportEnableSettings.gui,
+            permissions.dataExportEnableApp.gui,
+        ])
+            .then(userProperties => {
+                user = userProperties;
+                cy.login(user.username, user.password, { path: TopMenu.settingsPath, waiter: SettingsPane.waitLoading });
+            });
+    });
+
+    after('deleting user', () => {
+        Users.deleteViaApi(user.userId);
```

Comment:
Please add deleting the created mapping profile

Reply:
done